---
layout: paper
title: Multitable Methods for Microbiome Data Integration
authors: Kris Sankaran, Susan P Holmes
image: /assets/img/multitable_microbiome.png
abstract: The simultaneous study of multiple measurement types is a frequently encountered problem in practical data analysis. It is especially common in microbiome research, where several sources of data—for example, 16s-rRNA, metagenomic, metabolomic, or transcriptomic data–can be collected on the same physical samples. There has been a proliferation of proposals for analyzing such multitable microbiome data, as is often the case when new data sources become more readily available, facilitating inquiry into new types of scientific questions. However, stepping back from the rush for new methods for multitable analysis in the microbiome literature, it is worthwhile to recognize the broader landscape of multitable methods, as they have been relevant in problem domains ranging across economics, robotics, genomics, chemometrics, and neuroscience. In different contexts, these techniques are called data integration, multi-omic, and multitask methods, for example. Of course, there is no unique optimal algorithm to use across domains—different instances of the multitable problem possess specific structure or variation that are worth incorporating in methodology. Our purpose here is not to develop new algorithms, but rather to 1) distill relevant themes across different analysis approaches and 2) provide concrete workflows for approaching analysis, as a function of ultimate analysis goals and data characteristics (heterogeneity, dimensionality, sparsity). Towards the second goal, we have made code for all analysis and figures available online at https://github.com/krisrs1128/multitable_review.
journal: Frontiers in Genetics. August 28, 2019. https://doi.org/10.3389/fgene.2019.00627
journalurl: https://www.frontiersin.org/articles/10.3389/fgene.2019.00627/full
pdfurl: /assets/papers/fgene-10-00627.pdf
code: https://github.com/krisrs1128/multitable_review
---
